packages/console/src/pages/Connectors/components/Guide/index.module.scss

```diff
@@ -42,15 +42,15 @@
       max-width: 800px;
       min-width: 400px;
       overflow-y: auto;
-      margin: _.unit(2) 0 _.unit(6) 0;
     }
 
     .readme {
+      display: flex;
+      flex-direction: column;
       background-color: var(--color-layer-1);
       border: 1.5px solid var(--color-focused-variant);
       border-radius: 16px;
-      margin-right: _.unit(6);
-      overflow: hidden;
+      margin: _.unit(2) _.unit(6) _.unit(6) 0;
```

Comment:
use padding for additional space? i think the bottom padding can be 4 units larger

Reply:
This margin is used for the outer space of the readme component.
<img width="826" alt="image" src="https://user-images.githubusercontent.com/10806653/226108652-a1baaac2-bf78-4bee-a7c9-c46197f43a58.png">

Reply:
Updated
<img width="899" alt="image" src="https://user-images.githubusercontent.com/10806653/226109107-191b4b8a-d2a9-41bd-b16c-c7715958cf24.png">